test/integ/cli/test_cli_manifest_download.py

```diff
@@ -198,10 +198,13 @@ def test_manifest_download_job(
 
                 # Create a list of files we know should be in the input paths.
                 files: List[str] = [path.path for path in manifest.paths]
-                assert os.path.join("inputs", "textures", "brick.png") in files
-                assert os.path.join("inputs", "textures", "brick.png") in files
-                assert os.path.join("inputs", "scene.ma") in files
+                assert "inputs/textures/brick.png" in files
+                assert "inputs/textures/brick.png" in files
+                assert "inputs/scene.ma" in files
 
+    @pytest.mark.skip(
```

Comment:
I cannot reproduce this failure on my local setup. I will debug the test infrastructure to find out more. Maybe missing a permission still.